ironmon_tracker/data/RandomizerLog.lua

```diff
@@ -1,5 +1,7 @@
 -- A collection of tools for viewing a Randomized Pokémon game log
-RandomizerLog = {}
+RandomizerLog = {
+	currentLog = nil,
```

Comment:
Can you add a comment to explain what this variable is? At first I thought it held the parsed log information for the currently loaded log file, but then I remembered this info is already kept as part of `RandomizerLog.Data`. If this is meant to be something else, can you add a description here for that.

Reply:
This variables just holds the log path of the previously loaded log, or is nil if no log has been loaded. Added this to distinguish between previous log and log for current run when checking if log data exists to load into the overlay.

Its used in LogOverlay in this block at line 474-477

```	
-- Check for what log we're trying to display, and if it's already been parsed
	if RandomizerLog.loadedLogPath ~= logpath then
		RandomizerLog.Data = {}
		RandomizerLog.loadedLogPath = logpath
	end
```
added a comment to explain this and renamed the variable